controller/config.go

```diff
@@ -995,6 +1004,16 @@ func Validate(c Config) error {
 		}
 	}
 
+	if v, ok := c[LoginTokenRefreshURL].(string); ok {
+		u, err := url.Parse(v)
+		if err != nil {
+			return errors.Annotate(err, "invalid login token refresh URL")
+		}
+		if u.Scheme == "" || u.Host == "" {
```

Comment:
They should also always use "https" as the scheme. Makes development more difficult, but here its important.